doc/source/building/cross_compilation.rst

```diff
@@ -4,11 +4,20 @@ Cross compilation
 Cross compilation is a complex topic, we only add some hopefully helpful hints
 here (for now). As of May 2023, cross-compilation based on ``crossenv`` is
 known to work, as used (for example) in conda-forge. Cross-compilation without
-``crossenv`` requires some manual overrides, as done in (for example) Void Linux.
+``crossenv`` requires some manual overrides. You instruct these overrides by
+passing options to ``meson setup`` either manually, or via `meson-python`_.
 
-Please see `Meson's documentation on cross compilation
-<https://mesonbuild.com/Cross-compilation.html>`__
-for details on Meson's support for cross-compilation.
+.. _meson-python: https://meson-python.readthedocs.io/en/latest/how-to-guides/meson-args.html
+
+Distributions that are known to successfully cross compile SciPy are:
```

Comment:
It may be useful to edit this sentence like:
```
... compile SciPy, all using ``pypa/build`` via ``python -m build``, are:
```
and then remove it from the list below.

Reply:
> It may be useful to edit this sentence like:
> 
> ```
> ... compile SciPy, all using ``pypa/build`` via ``python -m build``, are:
> ```
> 
> and then remove it from the list below.

I agree!